src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -33,6 +33,22 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
     background-color: transparent;
     cursor: pointer;
   }
+
+  &::after {
+    content: '';
+    position: absolute;
+    height: $range-track-height;
+    width: 100%;
+    background-image: linear-gradient(
+      to right,
+      var(--p-surface-inverse-foreground),
+      var(--p-surface-inverse-foreground) 50%,
```

Comment:
@danrosenthal @tmlayton I'm not sure if this is the correct color to use. 

For reference this is the Figma example:
<img width="859" alt="Screen Shot 2019-11-21 at 4 53 32 PM" src="https://user-images.githubusercontent.com/22782157/69379970-7ade7280-0c7f-11ea-9bdb-79110f9451dc.png">

Though the `surface` colors resemble a darker shade:

![image](https://user-images.githubusercontent.com/22782157/69380089-b5e0a600-0c7f-11ea-9d29-92a91facf3db.png)

I also considered the `surface border` colors since they're slightly lighter:

<img width="852" alt="Screen Shot 2019-11-21 at 4 56 59 PM" src="https://user-images.githubusercontent.com/22782157/69380212-f4766080-0c7f-11ea-85bf-e5fcd65b415e.png">

I'm assuming we should closely stick to the naming convention and tweak colors later, curious what y'all think?

Reply:
> I'm assuming we should closely stick to the naming convention and tweak colors later, curious what y'all think?

Yes, stick with the most fitting name and tweak colors later. Although, surface foreground won’t change much. Maybe `--p-surface-inverse-foreground-subdued` (soon to be called `--p-surface-foreground-subdued-inverse`)? Maybe a variant in the neutral role or a disabled variant (since it is kind of like the inactive part of the slider? Also not opposed to a new variant as well if something makes sense.